docs/api/supervisor/models.md

```diff
@@ -306,3 +306,14 @@ Response only fields will be in responses but cannot be included in requests.
 | ---------- | ------- | ---------------------------------------------- |
 | type       | string  | Type of error that occurred                    |
 | message    | string  | Human-readable description of what went wrong  |
+
+## User
+
+| key        | type    | description                                                   |
+| ---------- | ------- | ------------------------------------------------------------- |
+| username   | string  | Username used to login                                        |
+| name       | string  | Name of the user                                              |
+| is_owner   | boolean | Is the user the owner                                         |
+| is_active  | boolean | Is the user active                                            |
+| local_only | boolean | Can the user login outside the LAN                            |
```

Comment:
It seems that local_only really means localhost only (see https://github.com/home-assistant/core/blob/2024.3.0b3/homeassistant/components/http/auth.py#L92-L93). So:

```suggestion
| local_only | boolean | Can the user login from the network (e.g. via http)           |
```